core/chains/evm/client/rpc_client.go

```diff
@@ -75,6 +75,8 @@ var (
 			float64(8 * time.Second),
 		},
 	}, []string{"evmChainID", "nodeName", "rpcHost", "isSendOnly", "success", "rpcCallName"})
+	errSubscribeFilterLogsNotAllowedWithoutWS = errors.New("SubscribeFilterLogs is not allowed without ws url")
```

Comment:
Are we reusing these variables across multiple functions? If not, it might make sense to define them inline or make them public and use in tests